### PR TITLE
build: landing page for ecomm

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
Build landing page for ecommerce app and refactored navbar css .

### CODE REVIEW

File: styles/reset.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+:root {
+    --ecomm-color-primary: #0000b7;
+    --ecomm-color-secondary:#ffffff;
+    --ecomm-color-border:#000000;
+    --ecomm-color-navbar:rgb(247 251 255);
```
Comment (rutu14) [inline styles/reset.css]: Use hex code to keep the consistency.

**Review by Rockyraven — APPROVED** [review]
All good to me ! Can merge

File: styles/landing-page.css [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+}
+.social-links-list {
+    display: flex;
+    justify-content: space-around;
+}
+.footer-col ul li a{
+    color: var(--ecomm-color-border);
+}
+li{
+    margin: 10px 0;
```
Comment (rutu14) [inline styles/landing-page.css:117]: Use class selector, so that it doesn't apply it globally.

**Review by rutu14 — APPROVED** [review]

**Review by arjun-computer-geek — APPROVED** [review]
Looks Good To Me

**Pull request merged**